core-plugins/pom.xml

```diff
@@ -222,6 +222,7 @@
               co.cask.hydrator.plugin.validator.*;
               org.apache.avro.mapreduce;
               parquet.avro.*;
+              parquet.hadoop.*;
```

Comment:
Without this dependency, when reading from the TPFSParquetSource, we will get ClassNotFoundException for ParquetInputSplit class.